models/issue.go

```diff
@@ -1260,6 +1260,30 @@ func Issues(opts *IssuesOptions) ([]*Issue, error) {
 	return issues, nil
 }
 
+// CountIssues number return of issues by given conditions.
+func CountIssues(opts *IssuesOptions) (int64, error) {
+	sess := x.NewSession()
+	defer sess.Close()
+
+	var cnt int64
+	countsSlice := make([]*struct {
+		RepoID int64
+		Count  int64
+	}, 0, len(opts.RepoIDs))
+
+	sess.GroupBy("issue.repo_id").
+		Select("issue.repo_id AS repo_id, COUNT(*) AS count").Table("issue")
```

Comment:
Why the GroupBy? Why not just do simple Count?

Reply:
Yes, I want to use function of simple count.
But It function is included bug when MSSQL.
So I reluctantly did use the GroupBy. -> https://github.com/go-gitea/gitea/pull/12275/commits/3f099650f75e7723b6d19616dea464d4e0840e85

Reply:
Hmm that's interesting and looks like a potential bug in Xorm. However could you try:

```suggestion
	sess.Select("COUNT(*) AS count").Table("issue")
```

Or if that doesn't work:

```suggestion
	sess.Select("COUNT(issue.id) AS count").Table("issue")
```

Reply:
It was try ;-D

Reply:
And the same problem arose.

Reply:
Sorry, This suggested code seems a bit different from what I tried.
I will check it tonight.

Reply:
@zeripath 
thank you for your suggest code. I updated.